Source/Chart.swift

```diff
@@ -521,30 +532,32 @@ open class Chart: UIControl {
 
         // horizontal axis at the bottom
         context.move(to: CGPoint(x: CGFloat(0), y: drawingHeight + topInset))
-        context.addLine(to: CGPoint(x: CGFloat(drawingWidth), y: drawingHeight + topInset))
+        context.addLine(to: CGPoint(x: CGFloat(drawingWidth + leftInset), y: drawingHeight + topInset))
         context.strokePath()
 
         // horizontal axis at the top
-        context.move(to: CGPoint(x: CGFloat(0), y: CGFloat(0)))
-        context.addLine(to: CGPoint(x: CGFloat(drawingWidth), y: CGFloat(0)))
+        context.move(to: CGPoint(x: CGFloat(drawingWidth + leftInset), y: CGFloat(0)))
+        context.addLine(to: CGPoint(x: CGFloat(drawingWidth + leftInset), y: CGFloat(0)))
         context.strokePath()
 
         // horizontal axis when y = 0
         if min.y < 0 && max.y > 0 {
             let y = CGFloat(getZeroValueOnYAxis(zeroLevel: 0))
             context.move(to: CGPoint(x: CGFloat(0), y: y))
-            context.addLine(to: CGPoint(x: CGFloat(drawingWidth), y: y))
+            context.addLine(to: CGPoint(x: CGFloat(drawingWidth + leftInset), y: y))
             context.strokePath()
         }
-
-        // vertical axis on the left
-        context.move(to: CGPoint(x: CGFloat(0), y: CGFloat(0)))
-        context.addLine(to: CGPoint(x: CGFloat(0), y: drawingHeight + topInset))
-        context.strokePath()
-
+        
+        if leftInset < 20 {
```

Comment:
What is this this [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming))?